### DISCUSSION TITLE
Include the top most Lists and Tags in the main menu Lists and Tags items

### DISCUSSION THREAD

**Discussion by amadeusp** [Kovah/LinkAce#457] [Ideas & Feature Requests]
I would find it practical to have the top most Tags and Lists directly accessible in the respective main menu drop-down items.

For example, the Lists drop-down content would then look like this:
- All Lists
- Top Listname 1
- Top Listname 2
- Top Listname 3
- ...
- Add List

**Comment by Kovah** [comment]
Thanks for trying LinkAce and suggesting new features.
Do you already have an idea what makes a list a "top list" so it could be included?

**Reply by amadeusp** [reply]
Thanks for making the app! :blush: Top lists (or tags), for me, would be like the most used ones, for example. I hope that makes sense.
